scripts/openroad/common/io.tcl

```diff
@@ -86,7 +86,16 @@ proc read {args} {
         keys {-override_libs}\
         flags {-multi_corner_libs}
 
-    if {[catch {read_db $::env(CURRENT_ODB)} errmsg]} {
+    if {
+        [catch {
+            if { [info exists ::env(IO_READ_DEF)] } {
+                read_lef $::env(MERGED_LEF)
+                read_def $::env(CURRENT_DEF)
+            } else {
+                puts "Reading odb $::env(CURRENT_ODB)"
```

Comment:
debug vestige?

Reply:
I left "Reading odb" because openroad doesn't print the input odb file while it does print the def and lef